uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -189,7 +189,7 @@ sub IsCandidateInfoValid {
     ####Get a list of files from the folder#####################
     #############Loop through the files#########################
     ############################################################
-    my $cmd = "find -path " . quotemeta($this->{'uploaded_temp_folder'}) . " -name '__MACOSX' -delete ";
+    my $cmd = "find " . quotemeta($this->{'uploaded_temp_folder'}) . " -name '__MACOSX' -delete ";
```

Comment:
the `-path` option of find is useful when you are looking for a filename containing `/` as the option `-name` gets rid of all `/` before the search. In this case, the string searched is `__MACOSX` so there is no need to use the option `-path` which does not exist in the default `find` command installed on Mac OS X systems.